src/components/cookie-banner/index.md.njk

```diff
@@ -29,95 +40,112 @@ Check with your organisation's privacy expert to see how data protection legisla
 
-## If the user’s device is not running JavaScript
+You’ll need to change the example cookie banner text if your service:
 
-If the user’s device is not running JavaScript, display a version of the cookie banner which tells them what to do in order to set or change their cookie preferences. Do not set any non-essential cookies, because the user has not consented to them.
+- allows third parties to set cookies (tell the user that both your organisation and other organisations will be setting cookies)
+- uses cookies for reasons other than collecting analytics information or remembering the user’s settings
 
-{{ example({group: "components", item: "cookie-banner", example: "no-js", html: true, nunjucks: true, open: false}) }}
+Keep the text as short as possible while making sure it’s an accurate description of how you use cookies. For example, if you use more than one ‘functional’ cookie and there’s not enough space to mention what each of them does, you could ask for permission to set cookies so ‘you can use as many of the service’s features as possible’.
 
-If your service only uses essential cookies, display a different version of the cookie banner to users whose devices are not running JavaScript.
+[See the cookies page for more advice on writing about cookies(/patterns/cookies-page/)].
 
-{{ example({group: "components", item: "cookie-banner", example: "essential-no-js", html: true, nunjucks: true, open: false}) }}
+### If you’re using essential cookies and analytics cookies
 
-## Getting consent from users who are not running JavaScript
+You can use this example text for a service which sets essential and analytics cookies - set by your organisation to collect analytics information about how well the service is performing.
 
-[Some users will not be running JavaScript on their devices](https://gds.blog.gov.uk/2013/10/21/how-many-people-are-missing-out-on-javascript-enhancement/). If you’re using non-essential cookies and want to get consent from these users, you’ll need an approach that does not depend on JavaScript.
+{{ example({group: "components", item: "cookie-banner", example: "default", html: true, nunjucks: true, open: false}) }}
 
-Show the replacement message confirming that the user has accepted or rejected cookies when they submit the form and the page has reloaded.
+### If you’re using more than one type of non-essential cookie
 
-Here’s an example showing how it should work.
+You can use this example text for a service that set:
+- essential cookies
+- analytics’ cookies
+- functional cookies to remember the user’s settings but are not essential
 
-{{ example({group: "components", item: "cookie-banner", example: "server-side", html: true, nunjucks: true, open: true}) }}
+{{ example({group: "components", item: "cookie-banner", example: "multiple-cookies", html: true, nunjucks: true, open: false}) }}
 
 ## Creating a cookies page
```

Comment:
Small thing but noticed we refer to this as 'cookies page' and 'cookie page' in different places

Reply:
Nope you're right - will change to match the name of our pattern.
Honestly, I'm not a fan of the inconsistency between cookie banner and cookies page, but I'm sure there's a reason for it!

Reply:
Okay, this is a bit trickier than I thought - the cookies page pattern has this inconsistency as well, will tackle these in our final pass at this.

Reply:
Commit done to replace all instances of "cookie page" to "cookies page"